go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/kserve/rest-proxy
 
-go 1.18
```

Comment:
this will require updated dependencies in `go.sum` by running `go mod tidy -compat=1.19`